test/elm/clinical/clinical-test.ts

```diff
@@ -186,6 +214,29 @@ describe('ExpandValueset', () => {
     ];
     received.should.eql(expected);
   });
+
+  it('expand valueset through function call', async function () {
+    const received: any = await this.expandPassedVS.exec(this.ctx);
+    const expected: Code[] = [new Code('F', '2.16.840.1.113883.18.2', 'HL7V2.5', undefined)];
+    received.should.eql(expected);
+  });
+
+  it('should return error for null cast valueset passed to function', async function () {
+    this.expandNullVS
```

Comment:
This situation should actually return `null` as the [spec](https://cql.hl7.org/09-b-cqlreference.html#expandvalueset) says "If the argument is null, the result is null."